vivisect/analysis/generic/pointers.py

```diff
@@ -72,12 +72,12 @@ def analyze(vw):
             try:
                 pname = vw.getName(ptr)
                 if pname is not None:
-                    logger.info('skipping renaming of ptr 0x%x (currently: %r)', ptr, pname)
+                    logger.debug('skipping renaming of ptr 0x%x (currently: %r)', ptr, pname)
```

Comment:
ok, now i'm confused... you want *more* logging?
or was this an accidental copy-pasta error?